src/engine/renderer/tr_image.cpp

```diff
@@ -2496,44 +2513,36 @@ static void R_CreateDepthRenderImage()
 		return;
 	}
 
-	int  width, height, w, h;
-
-	width = glConfig.vidWidth;
-	height = glConfig.vidHeight;
-
-	w = (width + TILE_SIZE_STEP1 - 1) >> TILE_SHIFT_STEP1;
-	h = (height + TILE_SIZE_STEP1 - 1) >> TILE_SHIFT_STEP1;
-
-	imageParams_t imageParams = {};
-	imageParams.bits = IF_NOPICMIP | IF_RGBA32F;
-	imageParams.filterType = filterType_t::FT_NEAREST;
-	imageParams.wrapType = wrapTypeEnum_t::WT_ONE_CLAMP;
+	{
```

Comment:
Useless braces?

Reply:
It makes things easier to read and keep variables in a local scope. Those blocks could be dedicated functions. Other blocks now just do things the same way, even sharing the same variable names, but in their own dedicated block.